cmake/projects/CURL/ci/matrix.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
Can you remove these custom CI files @ned14? We can track failures here: https://cpp-pm.github.io/hunter/